pkg/metrics/metrics.go

```diff
@@ -57,6 +58,16 @@ func (f Family) String() string {
 	return b.String()
 }
 
+// MetricType represents the type of a metric e.g. a counter. See
+// https://prometheus.io/docs/concepts/metric_types/.
+type MetricType string
+
+// MetricTypeGauge defines a Prometheus gauge.
+var MetricTypeGauge MetricType = "gauge"
```

Comment:
If we don't use a type alias, we could use constants instead, but I am not sure whether that has a real performance impact.